repository-data/application/src/main/resources/hcm-config/namespaces/website/biography.yaml

```diff
@@ -47,8 +53,14 @@ definitions:
             wicket.id: ${cluster.id}.left.item
           /prevpositions:
             /cluster.options:
-              ckeditor.config.appended.json: '{toolbar: [{ name: ''summarytoolbar'',  items:
-                [''Link'',''PickInternalLink'',''Source''] }] }'
+              ckeditor.config.appended.json: '{toolbarGroups: [{ name: ''document''}],
+                extraPlugins: ''balloonpanel,a11ychecker'', allowedContent: ''h2 h3
```

Comment:
just wondering if the ckeditor definitions in this file need to include the latest abbr added recently?

Reply:
yes - why not - I will add 'abbr' here

Reply:
solved by: https://github.com/NHS-digital-website/hippo/commit/4b1e9ee5a5497df9cca0f908a247b4bac8ad4d1a